emba.sh

```diff
@@ -378,14 +398,39 @@ main()
 
   echo
 
+  if [[ "$UPDATE" -eq 1 ]]; then
+    print_output "[*] EMBA update starting ..."
+    git pull
+    EMBA="$INVOCATION_PATH" FIRMWARE="$FIRMWARE_PATH" LOG="$LOG_DIR" docker pull embeddedanalyzer/emba
+    exit 0
```

Comment:
because it is in the help ;) No, because it updates the emba repo and so you need to restart EMBA

Reply:
Okey, I am not the biggest fan of this behavior, because EMBA did update the container and run afterwards the analysis in the past. But I tested `sudo ./emba.sh -U` and it works, so I can live with that :)